manifests/backends/postgresql.pp

```diff
@@ -81,9 +81,13 @@
   }
 
   if $::powerdns::backend_create_tables {
+    $password_hash = $db_password ? {
+      Undef   => undef,
+      default => postgresql_password($::powerdns::db_username, $_db_password),
```

Comment:
since ~2021 the postgresql module doesn't support undef passwords, so I had to patch this.